README.md

```diff
@@ -25,11 +24,11 @@ sum := func(init, seq) {
     return init
 }
 
-n := sum(0, [1, 2, 3])   // == 6
-s := sum("", [1, 2, 3])  // == "123"
+println(sum(0, [1, 2, 3]))   // "6"
+println(sum("", [1, 2, 3]))  // "123"
```

Comment:
fmt.println

Reply:
The playground runtime has its own `println` builtin function (that writes to internal buffer instead of stdout), and, "fmt" module is disabled. You can try it out in the link: https://tengolang.com/?s=3243fd27cd518aef287d16d59349e15a474fd76c

Reply:
Then perhaps a comment explaining that the code would only work on playground? I can imagine someone installing tengo binary and following the readme to try println and failing.